.github/workflows/tests.yml

```
@@ -63,6 +63,7 @@ jobs:
       - name: Install dependencies
         run: |
           pipenv install --dev --deploy --system
+          pip install "./site[$EXTRAS]"
```

```suggestion
          pip install "./site[tests]"
```

minor: since we're in the tests CI, I think that's reasonable to hardcode (and ideally we don't want to add any extras in the future)

---

While I do agree, the `$EXTRAS` variable is already defined and it includes the search extra (opensearch). At the moment the are not being used (I think), but the `run-tests.sh` script boots up the containers anyway. options:
a) simplify tests run, and remove containers etc. However, this would make it inconsistent with other modules. It would reduce running time (not much though)
b) keep extras as is

If a), I would suggest to be tackeld in a different issue.